kernel/src/main/scala/cats/kernel/std/option.scala

```diff
@@ -0,0 +1,68 @@
+package cats.kernel
+package std
+
+package object option extends OptionInstances
+
+trait OptionInstances extends OptionInstances1 {
+  implicit def optionOrder[A: Order] = new OptionOrder[A]
+  implicit def optionMonoid[A: Semigroup] = new OptionMonoid[A]
```

Comment:
FWIW `left.orElse(right)` is exposed through the `SemigroupK[Option]` instance (not in kernel).

If we _do_ provide these other instances (I don't necessarily see a strong need for them), I guess that `std` seems like a more natural place to me than the `Monoid` companion object. We probably have lots of types with alternative type class instances, and putting those with the other instances for those types seems like a more modular approach to me than shoving all of them into the corresponding type class companion objects (especially once you start talking about type class hierarchies). Just my 2 cents.